justfile

```diff
@@ -83,7 +83,7 @@ upgrade env package="": virtualenv
     #!/usr/bin/env bash
```

Comment:
I found the comment above here confusing. Github won't let me comment on it directly or suggest a change because it's not part of the diff. It reads as though "all by default" applies to the dev/prod distinction rather than the packages, but would be fine if it read "specify package to upgrade single package, all by default". Would you mind changing it while you're here?

Reply:
Oh yes I see what you mean, fixed 👍